figures/coronagraph_starfield.py

```diff
@@ -0,0 +1,51 @@
+import os
+import matplotlib.pyplot as plt
+import sunpy.map
+from sunpy.coordinates import get_body_heliographic_stonyhurst, frames
+
+# look up stars in a star catalog
+from astroquery.vizier import Vizier
+from astropy import coordinates
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy_paper import data_dir
+
+f = '2014_05_15__07_54_00_005__STEREO-A_SECCHI_COR2_white-light.jp2'
+
+path = os.path.join(data_dir, f)
+map1 = sunpy.map.Map(path)
+
+# finding the correct pointing location for the Sun as seen from STEREO
+sun = SkyCoord(0*u.deg, 0*u.deg, 0*u.km, obstime=map1.date,frame=frames.HeliographicStonyhurst, observer='Earth')
+stereo_icrs = map1.observer_coordinate.transform_to('icrs')
+sun_icrs = sun.transform_to('icrs')
```

Comment:
You can replace this with a call to http://docs.astropy.org/en/stable/api/astropy.coordinates.get_body_barycentric.html#astropy.coordinates.get_body_barycentric